README.md

```diff
@@ -1,6 +1,10 @@
 # drb-inland-salinity-ml
 This repository contains a pipeline for data gathering, processing, modeling, and visualizing results for machine learning models that predict salinity in inland reaches of the Delaware River Basin (DRB).
 
+# Building the pipeline with S3 storage
+To build the pipeline with the S3 storage, you will need to provide AWS credentials. We have been doing this using the [`saml2aws` tool](https://github.com/Versent/saml2aws). See also @amsnyder's [post on this](https://github.com/amsnyder/s3_demo/blob/main/usgs_access.md). Note - when you do `saml2aws login` make sure you choose the `gs-chs-wma-dev` option because that is where the S3 bucket is. With those credentials in the `~/.aws/credentials` file, R should be able to use them to upload/download to the inland salinity bucket.
```

Comment:
I did not use the .Renviron file. I think I tried, but it didn't work. PAWS reads the `~/.aws/credentials` file and gets the credentials from there. In my `credentials` file, I have the following fields which were generated by `saml2aws`

```
aws_access_key_id
aws_secret_access_key
aws_session_token
aws_security_token
x_principal_arn
x_security_token_expires
```

Reply:
The way I've done it, R needs to be able to read the `~/.aws/credentials` file. If running R locally or in Singularity, that is not a problem. If running through docker, I think that that file will have to be included in a volume (`-v`) so that R can access that file. I have not experimented with that yet.